README.md

```diff
@@ -34,6 +34,12 @@ If you are a Mac OS X user, you can use [Homebrew](https://brew.sh):
 brew install terraform-docs
 ```
 
+Windows users can install using [chocolatey](https://www.chocolatey.org):
+
+```
+choco install terraform-docs
```

Comment:
I'm not familiar with Chocolatey, do we need to publish the binary to their system for this to work (something like brew)?

Reply:
Yes, it works very similar to brew. And it looks like someone ("[DrFaust](https://chocolatey.org/profiles/DrFaust)"?) has already created a wrapper around the 8.2 installer in the releases folder here in github. (You can see their script in the "Files" section on the link below.) 

https://chocolatey.org/packages/Terraform-Docs

Reply:
Update: I was able to trace the package source code using the link on the Chocolatey website. The manifest file is a `.nuspec` file, and then there are two powershell scripts that do the remainder of the work. The author is @drfaust92 here on Github.

https://github.com/DrFaust92/chocolatey-packages/tree/master/terraform-docs

Reply:
Hey guys, I've added this package to choco(and closed my issue as I didn't notice this one)

I handled the the installation script, this is indeed like brew but for Windows.

Ill keep updating this as new versions are released.

Reply:
Thanks @aaronsteers for the update.
@DrFaust92 that is awesome. How do you feel to move the config you've been maintaining to this repo (as an official config) e.g. in `scripts/choco/` and keep maintaining it here (or may be we can even automate it on release)

Reply:
Sure, I've been looking into automating this either way.
I'll open a seperate PR after I think how should the access key to Choco can be handled.

Reply:
That's great. We can track that in another issue/PR of its own. I'm gonna merge this. Thanks folks.